app/src/main/res/drawable/gradient.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <gradient android:startColor="#E48BC34A"
+                android:endColor="#B71DC4DA"
+                android:type="linear"
+                android:angle="90"
+                />
```

Comment:
place this at the end of `android:angle="90"`

Reply:
remember to run a format code or reformat code, it should fix this kind of things. It is not always perfect, so be careful